src/pages/Chat/index.jsx

```diff
@@ -12,27 +12,27 @@ import Layout from './Chat.layout'
 const Chat = () => {
   const { id } = useParams()
   const history = useHistory()
-
-  if (!id) {
+  const packageId = JSON.parse(sessionStorage.getItem('packageId'))
+  if (!id || !packageId) {
```

Comment:
Is package id or package code?  how is this in the database? I think it is the more important

Reply:
package code is the string "test", and packageId is the UIID that in the DB is just ID